utils/orm_database/database.py

```diff
@@ -0,0 +1,354 @@
+from collections.abc import Sequence
+from datetime import datetime
+
+import discord
+from sqlalchemy import Row, select
+from sqlalchemy.ext.asyncio import (
+    AsyncEngine,
+    AsyncSession,
+    async_sessionmaker,
+    create_async_engine,
+)
+
+from config import DATABASE_URL
+
+from ..enums import InfractionsEnum, SettingsEnum, StatTypeEnum
+from ..logger import CustomLogger
+from .models import *
+
+
+class ORMDataBase:
```

Comment:
new database system, could be lifted out of the subordner when we sunset the ShortStoreDB